circle.yml

```diff
@@ -34,15 +36,17 @@ test:
     - npm run flow
 
 dependencies:
+  pre:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash
   cache_directories:
     - "~/downloads"
-
+    - ~/.yarn-cache
   override:
     - npm i -g jasonlaster/lerna
-    - lerna bootstrap
-    - ./bin/install-chrome
-    - ./bin/install-firefox
+    - yarn install
+    - docker pull jlongster/mochitest-runner
```

Comment:
This shouldn't be needed as the `./bin/update-docker` command below should be doing this docker pull.